{
  "uid" : "a65efa728c24a860",
  "name" : "test_transfer_zero_tokens",
  "fullName" : "tests.brownie.token.tests.test_transferFrom#test_transfer_zero_tokens",
  "historyId" : "712015a3b92bfa1e855577e4791f38fb",
  "time" : {
    "start" : 1642031189673,
    "stop" : 1642031189673,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: connection already closed",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7fae302657f0>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:725: in _make_transaction\n    raise VirtualMachineError(e) from None\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError({'code': -32000, 'message': 'connection already closed'}))\nexc = {'code': -32000, 'message': 'connection already closed'}\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n>               raise ValueError(exc[\"message\"]) from None\nE               ValueError: connection already closed\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:85: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "configure_neon_network",
    "time" : {
      "start" : 1642031167582,
      "stop" : 1642031167582,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "brownie_project",
    "time" : {
      "start" : 1642031168838,
      "stop" : 1642031184355,
      "duration" : 15517
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "accounts",
    "time" : {
      "start" : 1642031185279,
      "stop" : 1642031185605,
      "duration" : 326
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "setup_env",
    "time" : {
      "start" : 1642031185279,
      "stop" : 1642031185279,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "token",
    "time" : {
      "start" : 1642031188430,
      "stop" : 1642031188970,
      "duration" : 540
    },
    "status" : "broken",
    "statusMessage" : "ValueError: connection already closed\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 725, in _make_transaction\n    raise VirtualMachineError(e) from None\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 85, in __init__\n    raise ValueError(exc[\"message\"]) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "accounts::0",
    "time" : {
      "start" : 1642031256987,
      "stop" : 1642031256987,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transferFrom"
  }, {
    "name" : "host",
    "value" : "fv-az74-735"
  }, {
    "name" : "thread",
    "value" : "3663-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transferFrom"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 22,
        "skipped" : 0,
        "passed" : 67,
        "unknown" : 0,
        "total" : 89
      },
      "items" : [ {
        "uid" : "db11e36e66c58894",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/476//#testresult/db11e36e66c58894",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642028351281,
          "stop" : 1642028351281,
          "duration" : 0
        }
      }, {
        "uid" : "ff5a10bb0b073616",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/474//#testresult/ff5a10bb0b073616",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642025673426,
          "stop" : 1642025673426,
          "duration" : 0
        }
      }, {
        "uid" : "6fe3b3d5bc846bf7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/472//#testresult/6fe3b3d5bc846bf7",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642018239129,
          "stop" : 1642018239129,
          "duration" : 0
        }
      }, {
        "uid" : "d37db9b02fca48b4",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/471//#testresult/d37db9b02fca48b4",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642015605087,
          "stop" : 1642015605087,
          "duration" : 0
        }
      }, {
        "uid" : "903add5b6e2f33eb",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/470//#testresult/903add5b6e2f33eb",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642014231209,
          "stop" : 1642014231209,
          "duration" : 0
        }
      }, {
        "uid" : "a9dcc1dbbd9b2625",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/469//#testresult/a9dcc1dbbd9b2625",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642012912687,
          "stop" : 1642012912687,
          "duration" : 0
        }
      }, {
        "uid" : "add1156a562c0aa3",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/468//#testresult/add1156a562c0aa3",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642011423497,
          "stop" : 1642011423497,
          "duration" : 0
        }
      }, {
        "uid" : "7c1056dacee256d2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/466//#testresult/7c1056dacee256d2",
        "status" : "passed",
        "time" : {
          "start" : 1641998025516,
          "stop" : 1641998030549,
          "duration" : 5033
        }
      }, {
        "uid" : "dda52f08a0a166cc",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/465//#testresult/dda52f08a0a166cc",
        "status" : "passed",
        "time" : {
          "start" : 1641993787035,
          "stop" : 1641993792072,
          "duration" : 5037
        }
      }, {
        "uid" : "c32dbbbd8d9737b1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/464//#testresult/c32dbbbd8d9737b1",
        "status" : "passed",
        "time" : {
          "start" : 1641991041807,
          "stop" : 1641991046095,
          "duration" : 4288
        }
      }, {
        "uid" : "92dc057c36f958cf",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/463//#testresult/92dc057c36f958cf",
        "status" : "passed",
        "time" : {
          "start" : 1641986388323,
          "stop" : 1641986391947,
          "duration" : 3624
        }
      }, {
        "uid" : "a5afb1ac0ef5dbed",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/461//#testresult/a5afb1ac0ef5dbed",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895fe5e80>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641982478991,
          "stop" : 1641982478991,
          "duration" : 0
        }
      }, {
        "uid" : "75b091ab39d557dd",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/460//#testresult/75b091ab39d557dd",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189460db20>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641980268030,
          "stop" : 1641980268030,
          "duration" : 0
        }
      }, {
        "uid" : "f4aa8e6bf6d66a7d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/459//#testresult/f4aa8e6bf6d66a7d",
        "status" : "passed",
        "time" : {
          "start" : 1641975858373,
          "stop" : 1641975863057,
          "duration" : 4684
        }
      }, {
        "uid" : "3005d042c678f5dc",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/458//#testresult/3005d042c678f5dc",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189a842400>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974828100,
          "stop" : 1641974828100,
          "duration" : 0
        }
      }, {
        "uid" : "c0345b9ed68f43de",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/457//#testresult/c0345b9ed68f43de",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1896042940>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974206684,
          "stop" : 1641974206684,
          "duration" : 0
        }
      }, {
        "uid" : "12246877e5fc3214",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/456//#testresult/12246877e5fc3214",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641958012312,
          "stop" : 1641958012312,
          "duration" : 0
        }
      }, {
        "uid" : "a00cf81475feded2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/455//#testresult/a00cf81475feded2",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641952208416,
          "stop" : 1641952208416,
          "duration" : 0
        }
      }, {
        "uid" : "68a7f84894c32207",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/454//#testresult/68a7f84894c32207",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641950686117,
          "stop" : 1641950686117,
          "duration" : 0
        }
      }, {
        "uid" : "93538e124622a6b0",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/453//#testresult/93538e124622a6b0",
        "status" : "broken",
        "statusDetails" : "ValueError: ['Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument']",
        "time" : {
          "start" : 1641947815710,
          "stop" : 1641947815710,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "a65efa728c24a860.json",
  "parameterValues" : [ ]
}